js/impact.js

```diff
@@ -83,7 +83,7 @@ var GLPIImpact = {
     ACTION_EDIT_DEPENDS_COLOR           : 12,
     ACTION_EDIT_IMPACT_COLOR            : 13,
     ACTION_EDIT_IMPACT_AND_DEPENDS_COLOR: 14,
-
+    ACTION_EDIT_EDGE                    : 15,
```

Comment:
I think we need to take this new action into account in the `undo` and `redo` functions.